components/ContentPicker/SortableList.js

```diff
@@ -0,0 +1,49 @@
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
+import PickedItem from './PickedItem';
+
+const { useSelect } = wp.data;
```

Comment:
@tlovett1 We need to use the import syntax for all wordpress dependencies for the dependency extraction plugin to pull it in correctly.

Reply:
Didn't know that. Thanks!